core/templates/dev/head/exploration_editor/main_tab/StateEditor.js

```diff
@@ -97,6 +97,9 @@ oppia.controller('StateEditor', [
 
     $scope.openStateContentEditor = function() {
       if (editabilityService.isEditable()) {
+        if ($scope.isCurrentStateInitialState()) {
```

Comment:
Optional: if you're going to do a check here, maybe go all the way and define a function like $scope.isCurrentStateTheOnlyState() -- and ditto below? Not sure though; just putting it out there as a suggestion.